cspatients/views.py

```diff
@@ -185,6 +185,42 @@ def post(self, request):
         return Response(data, status=return_status)
 
 
+class MultiSelectView(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request):
+        valid = True
+        selected_items = []
+
+        def clean_and_split(str):
```

Comment:
I think this should be defined outside of this view and perhaps in `utils.py` in order to have better tested software.

Reply:
yeah thats just lazy, moved